Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM alpine:3.3
+
+MAINTAINER PRX <sysadmin@prx.org>
+
+RUN echo "http://nl.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories && \
```

Comment:
Will do.  And I've already added a staging RDS mysql instance, hooked up in ECS -> http://staging.cms.prx.tech

Much faster than tunneling out.